### ISSUE TITLE
As a devops specialist I want the data warehouse deployment to use its own terraform so we don't get blocked by terraform cloud during deployments

### ISSUE THREAD

**Issue by dleard** [bcgov/cas-data-warehouse#10]
We don't want to use the cas-shelf repo anymore for bucket provisioning because we sometimes get stuck in queues and have jobs timeout when provisioning storage buckets. We should look into how to provision our own buckets with the terraform api.

Todo:
- [ ] Explore how to use the terraform API to track our own state and provision the buckets in GCS
- [ ] Add the necessary terraform files and secrets needed to provision our buckets
- [ ] Add documentation on how this was done and the reasons behind not wanting to use terraform cloud

**Comment by rdromey** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Climate-Action-Secretariat-60ca4121764d710011481ca2/issues/bcgov/cas-data-warehouse/10?planning-poker) with Zenhub @andrea-williams @dleard @BCerki @JoshLarouche @Sepehr-Sobhani

**Comment by dleard** [comment]
duplicate of cas-registration 11. Closing
